src/index.html

```diff
@@ -21,7 +24,414 @@
+          </p>
+        </a>
+
+        <p class="card__product-code">Product code: 195434</p>
+
+        <div class="stars">
+          <div class="stars__star stars__star--active"></div>
+          <div class="stars__star stars__star--active"></div>
+          <div class="stars__star stars__star--active"></div>
+          <div class="stars__star stars__star--active"></div>
+          <div class="stars__star"></div>
+
+          <p class="stars__review-count">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-title">Price:</p>
+          <p class="card__price-amount">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          class="card__button"
+        >
+          BUY
+        </a>
+      </div>
+
+      <div class="card">
+        <a
+          class="card__link"
+          href="#"
+        >
+          <img
+            class="card__image"
+            src="./images/imac.jpeg"
+            alt="imac photo"
+          />
+          <p class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </p>
+        </a>
+
+        <p class="card__product-code">Product code: 195434</p>
+
+        <div class="stars">
+          <div class="stars__star stars__star--active"></div>
+          <div class="stars__star stars__star--active"></div>
+          <div class="stars__star stars__star--active"></div>
+          <div class="stars__star stars__star--active"></div>
+          <div class="stars__star"></div>
+
+          <p class="stars__review-count">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-title">Price:</p>
+          <p class="card__price-amount">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          class="card__button"
+        >
+          BUY
+        </a>
+      </div>
+
+      <div class="card">
+        <a
+          class="card__link"
+          href="#"
+        >
+          <img
+            class="card__image"
+            src="./images/imac.jpeg"
+            alt="imac photo"
+          />
+          <p class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </p>
+        </a>
+
+        <p class="card__product-code">Product code: 195434</p>
+
+        <div class="stars">
+          <div class="stars__star stars__star--active"></div>
+          <div class="stars__star stars__star--active"></div>
+          <div class="stars__star stars__star--active"></div>
+          <div class="stars__star stars__star--active"></div>
+          <div class="stars__star"></div>
+
+          <p class="stars__review-count">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-title">Price:</p>
+          <p class="card__price-amount">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          class="card__button"
+        >
+          BUY
+        </a>
+      </div>
```

Comment:
All card elements contain the same image and product details. If this is not intentional, consider updating each card with unique content to enhance the user experience.